app/src/main/res/layout-land/admin_auth_activity.xml

```diff
@@ -39,97 +39,74 @@
         app:titleTextColor="@color/white" />
     </com.google.android.material.appbar.AppBarLayout>
 
-    <FrameLayout
+    <View
+      android:id="@+id/admin_auth_toolbar_shadow_view"
       android:layout_width="0dp"
-      android:layout_height="0dp"
-      app:layout_constraintBottom_toBottomOf="parent"
+      android:layout_height="6dp"
+      android:background="@drawable/toolbar_drop_shadow"
       app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toBottomOf="@id/admin_auth_app_bar_layout">
-
-      <ScrollView
-        android:id="@+id/admin_auth_landscape_scrollView"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:overScrollMode="never"
-        android:scrollbars="none">
-
-        <androidx.constraintlayout.widget.ConstraintLayout
-          android:layout_width="match_parent"
-          android:layout_height="wrap_content"
-          android:paddingBottom="92dp">
```

Comment:
We still need this `padding_bottom`, you can introduce this in the root constraint_layout.

Reply:
Done.